starter_code/src/movies.js

```diff
@@ -1,9 +1,36 @@
 /* eslint no-restricted-globals: 'off' */
+var movies = require("./data.js");
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(array){
```

Comment:
This function is really large. It would have been best to break up this function into smaller more specialized functions and then create one main function to launch the program.